docs/data/material/components/about-the-lab/about-the-lab.md

```diff
@@ -19,24 +19,26 @@ For a component to be ready to move to the core, the following criteria are cons
 
 To install and save in your `package.json` dependencies, run one of the following commands:
 
-<!-- #default-branch-switch -->
+<!-- #npm-tag-reference -->
 
 <codeblock storageKey="package-manager">
 
 ```bash npm
-npm install @mui/lab @mui/material
+npm install @mui/lab@latest-v6 @mui/material@latest-v6
```

Comment:
We'll need to replace these occurrences as well, to not use latest-v6

Reply:
Done 👍🏼